DXMainClient/DXGUI/Multiplayer/CnCNet/GlobalContextMenu.cs

```diff
@@ -169,7 +169,16 @@ private void UpdateMessageBasedButtons()
             {
                 if (link == null)
                     return;
+
+#if NET48
                 Process.Start(link);
+#else
+                 _ = Process.Start(new ProcessStartInfo
+                {
+                    FileName = link,
+                    UseShellExecute = true
+                });
```

Comment:
See my earlier question on differentiating Process.Start code between .NET 4.8 and 6.

Reply:
See earlier response, I'll switch it to the new code.

Reply:
Code is updated